src/deployment.ts

```diff
@@ -87,6 +87,32 @@ export class Deployment {
         return Promise.resolve();
     }
 
+    @vsCommand('openshift.deployment.scale')
+    static async scalingPod(component: KubernetesObject): Promise<void> {
+        if (!component) {
+            return;
+        }
+        let count = '1';
+        count = await inputValue(`How many replicas would you like to scale ${component.kind}/${component.metadata.name}?`,
+            count,
+            false,
+            (value: string) => {
+                const trimmedValue = value.trim();
+                if (trimmedValue.length === 0) {
+                    return 'Scale value cannot be empty';
+                }
+                if (parseInt(trimmedValue, 10) < 0) {
```

Comment:
I am allowed to enter 0.001 into the wizard. When this is passed to `oc scale deployment`, it fails.

I think we should do one of:
- run `parseInt` on count before passing it to `oc scale deployment`
- Only allow integer values in the wizard

Reply:
Scale value now accepts only positive integer